modules/apps/frontend-js/frontend-js-react-web/src/main/resources/META-INF/resources/js/render.tsx

```diff
@@ -116,3 +120,34 @@ export default function render(
 		});
 	}
 }
+
+type Props = {
+	children: React.ReactNode;
+	spritemap: string;
+};
+
+function LiferayProvider({children, spritemap}: Props) {
+	const [accessibilityMenu] = useLiferayState(accessibilityMenuAtom);
+
+	const reducedMotion = useMemo(() => {
+		const setting = accessibilityMenu.find(
+			(setting) =>
+				setting.key === 'ACCESSIBILITY_SETTING_PREFERS_REDUCED_MOTION'
```

Comment:
This could probably be a constant exported by the `@liferay/accessibility-menu-web` module.